static/assets/app.js

```diff
@@ -27,16 +27,58 @@ app.constant("UTILS", {
   }
 });
 
+/**
+ * Convert factgory
+ */
+app.factory("Convert", function ($sce) {
```

Comment:
All the markup stuff we need from markdown is here which is unnecessary to load an entire library for it.